mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioWorker.java

```diff
@@ -629,9 +626,13 @@ public void run() {
                     int interestOps = channel.getInternalInterestOps();
```

Comment:
I think you need the following:
                    if (channel instanceof NioChildDatagramChannel) {
                        return;
                    }
It was there in the previous commits but got removed incorrectly. See the commit that removed it:
432b55c49e29ed9fed27bcd6a9e09ee5dd9696c7